src/deployment/deploy.py

```diff
@@ -397,8 +406,15 @@ def try_sp_create() -> None:
         ):
             set_app_audience(app.object_id, "AzureADMyOrg")
             url = "https://%s.azurewebsites.net" % self.application_name
+            api = "api://%s.azurewebsites.net" % self.application_name
+
+            if app.identifier_uris.contains(url):
+                identifier_uris = [url, api_url]
+            else:
+                identifier_uris = [api_url]
```

Comment:
We only need to update if the api url is not present

```suggestion
            if not app.identifier_uris.contains(api_url):
                identifier_uris = [url, api_url]
            else:
                identifier_uris = app.identifier_uris
```